src/snowflake/cli/_plugins/nativeapp/utils.py

```diff
@@ -96,3 +98,10 @@ def verify_no_directories(paths_to_sync: Iterable[Path]):
 def verify_exists(path: Path):
     if not path.exists():
         raise ClickException(f"The following path does not exist: {path}")
+
+
+def current_role() -> str:
```

Comment:
Feels like this belongs on SqlExecutor

Reply:
This was moved from src/snowflake/cli/_plugins/nativeapp/project_model.py, so I think moving it could be a refactor on its own

Reply:
Basically this is a variation on my feedback from the last PR where we want to make sure we're restricting the places where we implicitly use the cli context.

Reply:
Moved to SqlExecutor